app/src/main/java/de/danoeh/antennapod/fragment/preferences/SwipePreferencesFragment.java

```diff
@@ -36,6 +38,10 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
             new SwipeActionsDialog(requireContext(), FeedItemlistFragment.TAG).show(() -> { });
             return true;
         });
+        findPreference(PREF_SWIPE_HISTORY).setOnPreferenceClickListener(preference -> {
```

Comment:
The preference is missing from the XML file, that's why it crashes when opening the screen.